projects/authentication/docs/requesting-credentials.md

```diff
@@ -0,0 +1,26 @@
+# Requesting client credentials
+
+Before you can start making authenticated API requests, you need to register your integration with us so we can provide you with the necessary credentials.
+
+While we are working on a new self-service platform where you will be able to register your project for any API integration and automatically recieve client credentials in the future, the method to request your credentials currently varies per API as it is handled by multiple teams.
+
+## UiTPAS
+
+To use [UiTPAS API v4](https://docs.publiq.be/docs/uitpas), see its [quickstart guide](https://docs.publiq.be/docs/uitpas/d0748f47a3dba-quick-start) to request a client id and client secret via the linked Google Form.
+
+## UiTdatabank
+
+UiTdatabank APIs currently uses a custom authentication method based on API keys instead of client ids and secrets.
+
+While support for the standardized authentication methods documented in this space has been implemented, new client ids and secrets are only provided to a select few partners that are trying out these new authentication methods in UiTdatabank. In the future this will be opened up to all integrators.
+
```

Comment:
People who want to combine UiTPAS and Entry API, can follow the flow described in the UiTPAS quickstart guide.

As a sidenote: ledenbeheer.be has customers who have UiTPAS (they need both UPS and Entry), other clients aren't in UiTPAS yet (they only need Entry). Obviously they follow the same flow for all requests.

Reply:
I will mention it for UiTPAS integrators. But I'm not sure if clients that do not need UiTPAS should already start using client id & secret instead of the API key flow. Gezinsbond did it to start using client access tokens, but it had the unintended side-effect that their users could could not edit the events/places created with the client access tokens in UDB3 since then, while they expected that to keep working. Plus it could mean a lot of extra manual work to set up their clients.